docs/Getting Started/openSUSE/openSUSE Leap Root on ZFS.rst

```diff
@@ -1102,6 +1129,10 @@ Step 12: Final Cleanup
 #. Wait for the system to boot normally. Login using the account you
    created. Ensure the system (including networking) works normally.
 
+#. Configure locale: Find your locale from ``locale -a`` commands output then set it with following command::
```

Comment:
What's the purpose to move it here?

Reply:
Still actual question, I don't see a reason to do it after basic system configuration.